forms/aws/lambda/reliability/lib/notifyProcessing.js

```diff
@@ -0,0 +1,56 @@
+// Process request and format for Notify
+const { NotifyClient } = require("notifications-node-client");
+const convertMessage = require("markdown");
+
+async function sendToNotify(formSubmission, sendReceipt) {
+  const templateID = "92096ac6-1cc5-40ae-9052-fffdb8439a90";
+  const notify = new NotifyClient("https://api.notification.canada.ca", process.env.NOTIFY_API_KEY);
+  const emailBody = convertMessage(formSubmission);
+  const messageSubject = `${
+    formSubmission.form.emailSubjectEn
```

Comment:
Not related to this PR since I see titleEn was used previously - it looks like we only use the English values? Do we have an issue logged to improve this down the line?

Reply:
I believe we're waiting on research with the form administrators on how they want to receive responses.  Do we continue to send emails or do we transition to the online fetching of responses.